# PhDThesis

The supplementary tables for my PhD thesis

Supplementary Table 2.1: The initial screen of 28 genes from DNA extracted from the FFPE tumour samples of 32 women with HER2+ breast cancer

Supplementary Table 2.2: The frequency of the 6 SNPs in the study population.

Supplementary Table 2.3: Adjusted p-values for all the Tukey Honest Significant Difference tests and ANOVA models across the HER2 and the all the breast cancer cell lines


Supplementary Table 3.1: Information and accession numbers for ChIP-seq and RNA-seq of the 6 cell lines used in this study

Supplementary Table 3.2: The TCGA patients that are HER2+ with their ER status as determined from a previous study

Supplementary Table 3.3: The genes identified in a previous study as markers for regulation in the ER and HER2 pathways

Supplementary Table 3.4: The list of genes in the HER pathway according to KEGG, with their histone trimethylation status and corresponding gene expression

Supplementary Table 3.5: The list of genes in the Estrogen Signaling pathway according to KEGG, with their histone trimethylation status and corresponding gene expression (in FPKM).

Supplementary Table 3.6: The combined list of genes for the HER and Estrogen signaling pathways according to KEGG, with their corresponding histone trimethylation status and GRO-seq values in FPKM.

Supplementary Table 3.7: The complete differential expression analysis of genes downstream of the HER2 pathway, comparing the ER+ with the ER- HER2+ patients in the TCGA data, with the corresponding bivalent promoter status found in our 4 cell lines

Supplementary Table 3.8: The complete differential expression analysis of genes downstream of the estrogen signaling pathway, comparing the ER+ with the ER- HER2+ patients in the TCGA data, with the corresponding bivalent promoter status found in our 4 cell lines


Supplementary Table 4.1: The HER pathway according to KEGG, with the bivalent mark status and reCUT&RUN (re) for the control, E2, and trastuzumab treated groups, along with their corresponding gene expression (in FPKM).

Supplementary Table 4.2: The downstream targets that signify HER2 activation according to a previous study [21], with bivalent and reCUT&RUN (re) mark status for the control, E2, and trastuzumab experimental groups, along with their corresponding gene expression (in FPKM)

Supplementary Table 4.3: The ER signaling pathway according to KEGG, with the bivalent and reCUT&RUN (re)mark status for the control, E2, and trastuzumab treated groups and their corresponding gene expression (in FPKM)

Supplementary Table 4.4: The downstream targets that signify ER activation according to a previous study [21], with bivalent and reCUT&RUN (re) mark status for the control, E2, and trastuzumab experimental groups, along with their corresponding gene expression (in FPKM)

Supplementary Table 4.5: The list of genes found to be significantly differentially expressed (p.adj < 0.05) between the control and trastuzumab treated cells, in SKBR3, along with their mark status and their gene expression (in FPKM) 

Supplementary Table 4.6: The list of genes found to be significantly differentially expressed (p.adj < 0.05) between the control and E2 treated cells, in SKBR3, along with their mark status and their gene expression (in FPKM)

Supplementary Table 4.7: The list of genes found to be significantly differentially expressed (p.adj < 0.05) between the control and E2 treated cells, in BT474, along with their gene expression (in FPKM)

Supplementary Table 4.8: The list of genes found to be significantly differentially expressed (p.adj < 0.05) between the control and trastuzumab treated cells, in BT474, along with their gene expression (in FPKM)
